daml-script/export/src/main/scala/com/daml/script/export/Config.scala

```diff
@@ -30,6 +31,13 @@ final case class ExportScript(
     damlScriptLib: String,
 ) extends ExportType
 
+// This is a product rather than a sum, so that we can raise
```

Comment:
we could consider using that only during config parsing and switching to a different type afterwards but probably not worth the complexity.

Reply:
Agreed. I wish scopt had something like optparse-applicative's `Alternative` instance.